charts/px-backup/README.md

```diff
@@ -0,0 +1,87 @@
+# PX-Backup
+
+PX-Central is a unified, multi-user, multi-cluster management interface. Using PX-Backup, users can backup/restore Kubernetes clusters with PX-Backup.
+
+## Installing the Chart
+
+To install the chart with the release name `px-backup`:
+
+Add portworx/px-backup helm repository using command:
+```console
+$ helm repo add portworx https://raw.githubusercontent.com/portworx/helm/master/stable
+```
+
+Update helm repository:
+```console
+$ helm repo update
+```
+
+Search for portworx repo:
+```console
+$ helm search repo portworx
+```
+Output:
+```console
+NAME                    CHART VERSION   APP VERSION     DESCRIPTION                                       
+portworx/portworx       1.0.0                           A Helm chart for installing Portworx on Kuberne...
+portworx/px-backup      1.0.0                           A Helm chart for installing PX-Backup with PX-C...
+```
+
+Helm 3:
+```console
+$ helm install px-backup portworx/px-backup --namespace px-backup --create-namespace
+```
+
+Helm 2:
+```console
+$ helm install --name px-backup portworx/px-backup --namespace px-backup
+```
+
+## Uninstalling the Chart
+
+To uninstall/delete the `px-backup` chart:
+
+```console
+$ helm delete px-backup --namespace px-backup
+```
+OR
+```console
+$ kubectl delete ns px-backup
```

Comment:
if you delete the namesapce, Kubernetes will delete all the manifests. But does helm list still show the release?

Reply:
if we delete the namespace then will not see anything in helm list but global objects like clusterrole, clusterrolebinding and some ingress controller resources remains as it is.